gnd/src/main/java/com/google/android/gnd/repository/local/Edit.java

```diff
@@ -60,11 +60,19 @@ public static int toInt(@Nullable Edit.Type value) {
   @NonNull
   public abstract Type getType();
 
+  /**
```

Comment:
Should these changes be in this PR?

Reply:
These leaked in because either I expected to merge the previous PR before sending this one, or because I inadvertently merged the change in. Once PR #19 is merged this delta should disappear.